src/services/configServices/SettingsService.js

```diff
@@ -63,7 +63,7 @@ class SettingsService {
         mergedConfigContent.url !== "" &&
         !mergedConfigContent.url.startsWith("https://")
       ) {
-        mergedConfigContent.url = "https://" + mergedConfigContent.url
+        mergedConfigContent.url = `https://${mergedConfigContent.url}`
```

Comment:
I can take this since I introduced this line earlier. The `url` will only be a domain because it is enforced by the validators check:

https://github.com/isomerpages/isomercms-backend/blob/dd5de4a9bfb75f227ce8b40dacda45056126109b/src/validators/RequestSchema.js#L239

Reply:
gotcha, thanks!